services/cloudaccounts/k8s/k8s.go

```diff
@@ -55,6 +56,11 @@ type ImageAssuranceEnableRequest struct {
 	Enabled        bool   `json:"enabled"`
 }
 
+type ImageAccessRuntimeMonitorEnableRequest struct {
+	CloudAccountId string `json:"cloudAccountId"`
```

Comment:
why is the json here "cloudAccountId" and at other places its "k8sAccountId"? can we make it consistent?

Reply:
actually I took example from ImageAssurance in all places. 4 lines above, it has "cloudAccountId"

Reply:
yes, I see that.
I think we can take the opportunity and change both of them to be like the others. but it's up to you to decide.